packages/nextjs/test/src/common/CookieManager.spec.ts

```diff
@@ -51,31 +52,33 @@ describe('Cookie Manager', () => {
     expect(cookieValue[0]).toContain('HttpOnly');
     expect(cookieValue[0]).toContain('Secure');
     expect(cookieValue[0]).toContain('SameSite=None');
-    expect(cookieValue[0]).toContain(`Domain=${cookieDomain}`);
+    expect(cookieValue[0]).toContain(`Domain=${COOKIE_DOMAIN}`);
+    expect(cookieValue[0].length).toEqual(COOKIE_MAX_LENGTH);
 
     expect(cookieValue[1]).toContain(`${secondCookieName}=${secondCookieValue}`);
     expect(cookieValue[1]).toContain('HttpOnly');
     expect(cookieValue[1]).toContain('Secure');
     expect(cookieValue[1]).toContain('SameSite=None');
-    expect(cookieValue[1]).toContain(`Domain=${cookieDomain}`);
+    expect(cookieValue[1]).toContain(`Domain=${COOKIE_DOMAIN}`);
+    expect(cookieValue[1].length).toBeLessThan(COOKIE_MAX_LENGTH);
 
-    expect(firstCookieValue + secondCookieValue).toEqual(bigCookieValue);
+    expect(firstCookieValue + secondCookieValue).toEqual(LARGE_COOKIE_VALUE);
   });
 
   it('createEmptyCookies should accepts cookie names and create cookies with empty value and expiration date equals to now', () => {
-    const firstCookieName = CookieManager.getCookieName(1, cookieName);
-    const secondCookieName = CookieManager.getCookieName(2, cookieName);
-    const cookiesToRemove = [cookieName, firstCookieName, secondCookieName];
+    const firstCookieName = CookieManager.getCookieName(1, COOKIE_NAME);
+    const secondCookieName = CookieManager.getCookieName(2, COOKIE_NAME);
+    const cookiesToRemove = [COOKIE_NAME, firstCookieName, secondCookieName];
```

Comment:
Can we maybe improve a bit the tests:
1. The value that we check that is being contained are appearing multiple times so should we extract it to const and use it in every test?
2. Since this is the same check if the cookie value in the zero location contain can we create an array of values and test it in for each?